### PR TITLE
Update atoma-contracts.md

### PR DESCRIPTION
**Pull request by joehill7**

### CODE REVIEW

File: website/atoma-contracts.md [hunk 1]
```diff
@@ -10,33 +10,29 @@ We have a current implementation of the Atoma contract on Sui. This contract is
...
-7. `Echelon specification` - Upon 5., the Atoma contract specifies compute `echelons`. These can be thought of shards of the network. Compute across echelons should be as homogeneous as possible. Both in terms of compute and memory requirements, process time and determinism. 
-8. `Request submission` - Every request to the Atoma Network (processed by Atoma tokens) is submitted via the Atoma's contract. Moreover, requests are paid in `TOMA` token.
+3. `Accrue fees` - Nodes accrue fees, indexed in `TOMA` token, based on the number and the type of requests they process. Accrued fees can only be withdrawn two epochs later.
+4. `Subscribe to AI models and other forms of compute types` - Upon registration, nodes should specify which AI models these nodes subscribe too (i.e., which models do the nodes currently host). Once a node registers for a given set of models, it can't change these, unless it deregisters and registers a second type, specifying the new set of models.
```
Comment (jorgeantonio21) [inline website/atoma-contracts.md:14]: Nodes subscribe to

**Review by jorgeantonio21 — APPROVED** [review]
Lgtm

File: website/atoma-contracts.md (hunk 1 above)
Comment (jorgeantonio21) [inline website/atoma-contracts.md:14]: Nodes subscribe to * instead of Nodes subscribe too

**Review by joehill7 — COMMENTED** [review]
Changed too -> to

**Review by jorgeantonio21 — APPROVED** [review]
LGTM

**Pull request merged**